content/realtime/usage.textile

```diff
@@ -85,9 +85,21 @@ blang[ruby].
   ```
 
 blang[java].
-  The Realtime library for Java and Android is "hosted on Github":https://github.com/ably/ably-java and is downloadable as a JAR from "https://github.com/ably/ably-java/releases":https://github.com/ably/ably-java/releases or via "JCentre":https://bintray.com/ably-io/ably.
+  The Realtime library for Java and Android is "hosted on Github":https://github.com/ably/ably-java and can be used by adding one line to <span language="default">@build.gradle@ dependencies section.
 
-  Ensure the JAR is included in your classpath as follows:
+  For Java applications:
+
+  ```[java]
+  compile 'io.ably:ably-java:0.8.7'
+  ```
+
+  For Android apps:
+
+  ```[java]
+  compile 'io.ably:ably-android:0.8.7'
+  ```
+
+  Ensure the library is included in your classpath as follows:
```

Comment:
I believe the action here is to add some comments about replacing with the latest version?  Also, I assume now unfortunately https://github.com/ably/docs/pull/215#issuecomment-266667473 is incorrect.  If so, that's a huge pain, see https://github.com/ably/tutorials/pulls to get an idea why, that's 6 PRs to fix all the commit SHAs that are then referenced from https://github.com/ably/docs/pull/215#issuecomment-266667473